File: java/8.6/example7.java
import java.util.Scanner;

public class example7 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Integer number = 5;
        Boolean guessed = false;
        while (guessed == false) {
            System.out.println("Guess the number");
            Integer guess = Integer.parseInt(scanner.nextLine());
            if (guess == number) {
                guessed = true;
            }
        }
        System.out.println("You gussed correct");
    }
}

File: rust/src/8.6/example7.rs
fn main() {
    let number = 5;
    let mut gussed = false;
    while gussed == false {
        let mut guesss = String::new();
        guesss = "".to_string();
        println!("Guess the number");
        std::io::stdin().read_line(&mut guesss).unwrap();
        let guess = guesss.trim().parse::<i32>().unwrap();
        if guess == number {
            gussed = true;
            println!("You gussed correct")
        }
    }
}